code/modules/hydroponics/seeds.dm

```diff
@@ -90,14 +90,23 @@
 			genes += new /datum/plant_gene/reagent(reag_id, reagents_add[reag_id])
 		reagents_from_genes() //quality coding
 
+/obj/item/seeds/Destroy()
+	// No AS ANYTHING here, because the list/genes could have typepaths in it.
```

Comment:
(good comment!)

Reply:
good comment, but weird behavior...

Reply:
At this point, all the typepaths should be instances, not typepaths. You shouldn't have to do this because anything that calls Destroy() should be Initialized() first.

Reply:
One of the initialize args for seeds (nogenes) doesn't convert the paths:datums ...

I tried putting as anything in this loop, but some global list initializer runtimes because of this, it's cursed

Reply:
jesus

Reply:
you should definitely clear the list if there's no genes in that case in initialize

Reply:
I think the genes lists needs a good bit of work which IMO is a bit out of scope for this PR but if you want me to do it here then I can go ahead and do it

Reply:
I hate seedcode

Reply:
I do, too 😔 I'm working hard on making seed and botany code less "8-years-old trash"